fast64_internal/sm64/sm64_geolayout_classes.py

```diff
@@ -447,6 +448,35 @@ def to_c(self):
         return "GEO_BRANCH(" + ("1, " if self.storeReturn else "0, ") + geo_name + "),"
 
 
+class GeoLayoutBleed(BleedGraphics):
+    def bleed_geo_layout_graph(self, fModel, geo_layout_graph):
+        if not fModel.inline:
+            return
```

Comment:
This happens multiple times, but with `inline` removed these checks should be moved outside these classes.